app/models/profile.rb

```diff
@@ -0,0 +1,11 @@
+class Profile < ApplicationRecord
+  belongs_to :user
+
+  validates :name, presence: true
+  validates :cell_phone, presence: true
```

Comment:
Pode juntar nesses casos
```suggestion
  validates :name, :cell_phone, presence: true
```

Reply:
Resolvido